mungegithub/mungers/flakesync/cache.go

```diff
@@ -115,7 +121,9 @@ func NewCache(getFunc ResultFunc) *Cache {
 	c := &Cache{
 		byJob:             jobMap{},
 		flakeQueue:        flakeMap{},
+		expireList:        list.New(),
```

Comment:
I guess the memory usage of a `container/list` is sufficiently superior to a `slice` for this usage to justify the indirection? I don't care. PR looks fine.